### DISCUSSION TITLE
Img2Img - Load image batch node

### DISCUSSION THREAD

**Discussion by limitsnova** [WASasquatch/was-node-suite-comfyui#174] [General]
Hi there,
trying to use the WAS node, load image batch, connected to the image slot. But I have some errors.

I just need to batch a folder of .png into img2img to be processed with comfyUI

I am new to the system and any help is appreciated. Thank you.

![comfy](https://github.com/WASasquatch/was-node-suite-comfyui/assets/112591706/dba4660d-820f-44ec-b625-769bf8609967)

**Comment by WASasquatch** [comment]
Looks like you're using a very old version of WAS-NS. `clear_counter` isn't even part of it (or the entire suite), and the class starts on line 4920

Please update and reinstall your WAS-NS.
